src/components/tour-table.tsx

```diff
@@ -81,7 +80,7 @@ export class TourTable extends Component<TourTableProps, TourTableState> {
     } return false;
   }
 
-  editTour(data: any) {
+  editTour(data: string) {
```

Comment:
this is not a string, it is a json of the type Tour, you need to define that type, please put it into the mapStoreToProps